image builder missing ./tmp/.packageinfo

### Describe the bug

feig@host ~/openwrt-imagebuilder-ath79-generic.Linux-x86_64 $ make clean; make image PROFILE="tplink_archer-c7-v2" PACKAGES="ath10k-board-qca988x attendedsysupgrade-common base-files block-mount busybox ca-bundle cgi-io dmesg dnsmasq drill dropbear ethtool-full firewall4 fstools fwtool getrandom grep hostapd-mbedtls hostapd-utils htop ip-tiny -iw iw-full iwinfo kernel kmod kmod-ath10k-ct kmod-ath9k kmod-cfg80211 kmod-gpio-button-hotplug kmod-hwmon-core kmod-mac80211 kmod-nf-conntrack kmod-nf-conntrack6 kmod-nf-flow kmod-nf-log kmod-nf-log6 kmod-nf-nat kmod-nf-reject kmod-nf-reject6 kmod-nfnetlink kmod-nft-core kmod-nft-fib kmod-nft-nat kmod-nft-offload kmod-nls-base kmod-phy-ath79-usb kmod-random-core kmod-scsi-core kmod-slhc kmod-usb-core kmod-usb-ehci kmod-usb-ledtrig-usbport kmod-usb-storage kmod-usb2 libiwinfo-data libiwinfo20230701 libkmod libustream-mbedtls20201210 logd lsof luci-app-attendedsysupgrade luci-app-commands luci-app-firewall luci-app-opkg luci-app-uhttpd luci-app-upnp luci-base luci-mod-admin-full luci-mod-network luci-mod-status luci-mod-system luci-proto-ipv6 luci-theme-bootstrap luci-theme-openwrt-2020 miniupnpd-nftables mtd nano-full netifd nftables-json odhcp6c odhcpd-ipv6only openssh-sftp-server openwrt-keyring opkg procd procd-seccomp procd-ujail procps-ng procps-ng-free psmisc px5g-mbedtls rpcd rpcd-mod-file rpcd-mod-iwinfo rpcd-mod-luci rpcd-mod-rpcsys rpcd-mod-rrdns rpcd-mod-ucode sed strace stubby swconfig terminfo uboot-envtools ubox ubus ubusd uci uclient-fetch ucode ucode-mod-fs ucode-mod-html ucode-mod-math ucode-mod-nl80211 ucode-mod-rtnl ucode-mod-ubus ucode-mod-uci ucode-mod-uloop uhttpd uhttpd-mod-ucode urandom-seed urngd usign wget-ssl wireless-regdb zlib -iw -ppp -kmod-ppp -luci-mod-ppp -kmod-lib-zstd -kmod-fs-btrfs -wpad-basic-mbedtls -ppp-mod-pppoe -ppp-mod-pppox -ppp-mod-pppoa -ath10k-firmware-qca988x-ct kmod-random-core"
rm -rf /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/tmp /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79 /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/bin/targets/ath79/generic
make: TMPDIR value /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/tmp: No such file or directory
make: using default temporary directory '/tmp'
make: TMPDIR value /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/tmp: No such file or directory
make: using default temporary directory '/tmp'
make: TMPDIR value /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/tmp: No such file or directory
make: using default temporary directory '/tmp'
Building images for ath79 - TP-Link Archer C7 v2
Packages: ath10k-board-qca988x attendedsysupgrade-common base-files block-mount busybox ca-bundle cgi-io dmesg dnsmasq drill dropbear ethtool-full firewall4 fstools fwtool getrandom grep hostapd-mbedtls hostapd-utils htop ip-tiny iw-full iwinfo kernel kmod kmod-ath10k-ct kmod-ath9k kmod-cfg80211 kmod-gpio-button-hotplug kmod-hwmon-core kmod-mac80211 kmod-nf-conntrack kmod-nf-conntrack6 kmod-nf-flow kmod-nf-log kmod-nf-log6 kmod-nf-nat kmod-nf-reject kmod-nf-reject6 kmod-nfnetlink kmod-nft-core kmod-nft-fib kmod-nft-nat kmod-nft-offload kmod-nls-base kmod-phy-ath79-usb kmod-random-core kmod-scsi-core kmod-slhc kmod-usb-core kmod-usb-ehci kmod-usb-ledtrig-usbport kmod-usb-storage kmod-usb2 libiwinfo-data libiwinfo20230701 libkmod libustream-mbedtls20201210 logd lsof luci-app-attendedsysupgrade luci-app-commands luci-app-firewall luci-app-opkg luci-app-uhttpd luci-app-upnp luci-base luci-mod-admin-full luci-mod-network luci-mod-status luci-mod-system luci-proto-ipv6 luci-theme-bootstrap luci-theme-openwrt-2020 miniupnpd-nftables mtd nano-full netifd nftables-json odhcp6c odhcpd-ipv6only openssh-sftp-server openwrt-keyring opkg procd procd-seccomp procd-ujail procps-ng procps-ng-free psmisc px5g-mbedtls rpcd rpcd-mod-file rpcd-mod-iwinfo rpcd-mod-luci rpcd-mod-rpcsys rpcd-mod-rrdns rpcd-mod-ucode sed strace stubby swconfig terminfo uboot-envtools ubox ubus ubusd uci uclient-fetch ucode ucode-mod-fs ucode-mod-html ucode-mod-math ucode-mod-nl80211 ucode-mod-rtnl ucode-mod-ubus ucode-mod-uci ucode-mod-uloop uhttpd uhttpd-mod-ucode urandom-seed urngd usign wget-ssl wireless-regdb zlib kmod-random-core base-files busybox ca-bundle dnsmasq dropbear firewall4 fstools kernel kmod-ath10k-ct kmod-ath9k kmod-gpio-button-hotplug kmod-nft-offload kmod-usb-ledtrig-usbport kmod-usb2 libc libgcc libustream-mbedtls logd mtd netifd nftables odhcp6c odhcpd-ipv6only opkg procd procd-seccomp procd-ujail swconfig uboot-envtools uci uclient-fetch urandom-seed urngd

Package list missing or not up-to-date, generating it.

Building package index...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/Packages.gz
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/openwrt_core
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/Packages.gz
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/openwrt_base
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/Packages.gz
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/openwrt_kmods
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/Packages.gz
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/openwrt_luci
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/Packages.gz
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/openwrt_packages
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/routing/Packages.gz
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/openwrt_routing
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/routing/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/telephony/Packages.gz
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/openwrt_telephony
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/telephony/Packages.sig
Signature check passed.
Downloading file:packages/Packages
Updated list of available packages in /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/../../../../../../home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/dl/imagebuilder
Downloading file:packages/Packages.sig
Signature check passed.

Installing packages...
Installing libc (1.2.4-4) to root...
Installing libgcc1 (12.3.0-4) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/libgcc1_12.3.0-4_mips_24kc.ipk
Configuring libgcc1.
Configuring libc.
Installing kernel (5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd) to root...
Configuring kernel.
Installing ath10k-board-qca988x (20230804-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ath10k-board-qca988x_20230804-1_mips_24kc.ipk
Installing attendedsysupgrade-common (8) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/attendedsysupgrade-common_8_all.ipk
Installing libubox20230523 (2023-05-23-75a3b870-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libubox20230523_2023-05-23-75a3b870-1_mips_24kc.ipk
Installing libubus20230605 (2023-06-05-f787c97b-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libubus20230605_2023-06-05-f787c97b-1_mips_24kc.ipk
Installing libuci20130104 (2023-08-10-5781664d-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libuci20130104_2023-08-10-5781664d-1_mips_24kc.ipk
Installing libjson-c5 (0.17-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libjson-c5_0.17-1_mips_24kc.ipk
Installing libblobmsg-json20230523 (2023-05-23-75a3b870-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libblobmsg-json20230523_2023-05-23-75a3b870-1_mips_24kc.ipk
Installing rpcd (2023-07-01-c07ab2f9-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/rpcd_2023-07-01-c07ab2f9-1_mips_24kc.ipk
Installing rpcd-mod-rpcsys (2023-07-01-c07ab2f9-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/rpcd-mod-rpcsys_2023-07-01-c07ab2f9-1_mips_24kc.ipk
Installing base-files (1545-r24315-ae500e62e2) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/base-files_1545-r24315-ae500e62e2_mips_24kc.ipk
Installing libnl-tiny1 (2023-07-27-bc92a280-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libnl-tiny1_2023-07-27-bc92a280-1_mips_24kc.ipk
Installing ubusd (2023-06-05-f787c97b-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ubusd_2023-06-05-f787c97b-1_mips_24kc.ipk
Installing ubus (2023-06-05-f787c97b-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ubus_2023-06-05-f787c97b-1_mips_24kc.ipk
Installing jshn (2023-05-23-75a3b870-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/jshn_2023-05-23-75a3b870-1_mips_24kc.ipk
Installing netifd (2023-10-20-5590a80e-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/netifd_2023-10-20-5590a80e-2_mips_24kc.ipk
Installing jsonfilter (2018-02-04-c7e938d6-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/jsonfilter_2018-02-04-c7e938d6-1_mips_24kc.ipk
Installing usign (2020-05-23-f1f65026-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/usign_2020-05-23-f1f65026-1_mips_24kc.ipk
Installing openwrt-keyring (2022-03-25-62471e69-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/openwrt-keyring_2022-03-25-62471e69-2_mips_24kc.ipk
Installing ubox (2022-08-13-4c7b720b-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ubox_2022-08-13-4c7b720b-2_mips_24kc.ipk
Installing fstools (2023-02-28-bfe882d5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/fstools_2023-02-28-bfe882d5-1_mips_24kc.ipk
Installing fwtool (2019-11-12-8f7fe925-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/fwtool_2019-11-12-8f7fe925-1_mips_24kc.ipk
Installing block-mount (2023-02-28-bfe882d5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/block-mount_2023-02-28-bfe882d5-1_mips_24kc.ipk
Installing busybox (1.36.1-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/busybox_1.36.1-1_mips_24kc.ipk
Installing ca-bundle (20230311-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ca-bundle_20230311-1_all.ipk
Installing cgi-io (2022-08-10-901b0f04-21) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/cgi-io_2022-08-10-901b0f04-21_mips_24kc.ipk
Installing dmesg (2.39.2-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/dmesg_2.39.2-1_mips_24kc.ipk
Installing terminfo (6.4-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/terminfo_6.4-2_mips_24kc.ipk
Installing libncurses6 (6.4-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libncurses6_6.4-2_mips_24kc.ipk
Installing dnsmasq (2.89-6) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/dnsmasq_2.89-6_mips_24kc.ipk
Installing drill (1.8.3-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/drill_1.8.3-1_mips_24kc.ipk
Installing libatomic1 (12.3.0-4) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/libatomic1_12.3.0-4_mips_24kc.ipk
Installing libopenssl3 (3.0.12-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libopenssl3_3.0.12-1_mips_24kc.ipk
Installing libldns (1.8.3-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/libldns_1.8.3-1_mips_24kc.ipk
Installing dropbear (2022.82-5) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/dropbear_2022.82-5_mips_24kc.ipk
Installing ethtool-full (6.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ethtool-full_6.5-1_mips_24kc.ipk
Installing libmnl0 (1.0.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libmnl0_1.0.5-1_mips_24kc.ipk
Installing firewall4 (2023-09-01-598d9fbb-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/firewall4_2023-09-01-598d9fbb-1_mips_24kc.ipk
Installing libucode20230711 (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libucode20230711_2023-10-19-07c03173-1_mips_24kc.ipk
Installing ucode (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode_2023-10-19-07c03173-1_mips_24kc.ipk
Installing kmod-nfnetlink (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nfnetlink_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-reject (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-reject_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-reject6 (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-reject6_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-conntrack (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-conntrack_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-conntrack6 (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-conntrack6_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-nat (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-nat_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-log (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-log_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-log6 (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-log6_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-hash (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-hash_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-crc32c (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-crc32c_5.15.137-1_mips_24kc.ipk
Installing kmod-lib-crc32c (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-lib-crc32c_5.15.137-1_mips_24kc.ipk
Installing kmod-nft-core (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nft-core_5.15.137-1_mips_24kc.ipk
Installing kmod-nft-fib (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nft-fib_5.15.137-1_mips_24kc.ipk
Installing kmod-nf-flow (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nf-flow_5.15.137-1_mips_24kc.ipk
Installing kmod-nft-nat (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nft-nat_5.15.137-1_mips_24kc.ipk
Installing kmod-nft-offload (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nft-offload_5.15.137-1_mips_24kc.ipk
Installing libnftnl11 (1.2.6-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libnftnl11_1.2.6-1_mips_24kc.ipk
Installing jansson4 (2.14-3) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/jansson4_2.14-3_mips_24kc.ipk
Installing nftables-json (1.0.9-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/nftables-json_1.0.9-1_mips_24kc.ipk
Installing ucode-mod-fs (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode-mod-fs_2023-10-19-07c03173-1_mips_24kc.ipk
Installing ucode-mod-ubus (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode-mod-ubus_2023-10-19-07c03173-1_mips_24kc.ipk
Installing ucode-mod-uci (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode-mod-uci_2023-10-19-07c03173-1_mips_24kc.ipk
Package fstools (2023-02-28-bfe882d5-1) installed in root is up to date.
Package fwtool (2019-11-12-8f7fe925-1) installed in root is up to date.
Installing getrandom (2022-08-13-4c7b720b-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/getrandom_2022-08-13-4c7b720b-2_mips_24kc.ipk
Installing grep (3.8-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/grep_3.8-2_mips_24kc.ipk
Installing libpcre2 (10.42-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libpcre2_10.42-1_mips_24kc.ipk
Installing hostapd-mbedtls (2023-09-08-e5ccbfc6-5) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/hostapd-mbedtls_2023-09-08-e5ccbfc6-5_mips_24kc.ipk
Installing hostapd-common (2023-09-08-e5ccbfc6-5) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/hostapd-common_2023-09-08-e5ccbfc6-5_mips_24kc.ipk
Installing ucode-mod-nl80211 (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode-mod-nl80211_2023-10-19-07c03173-1_mips_24kc.ipk
Installing ucode-mod-rtnl (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode-mod-rtnl_2023-10-19-07c03173-1_mips_24kc.ipk
Installing ucode-mod-uloop (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode-mod-uloop_2023-10-19-07c03173-1_mips_24kc.ipk
Installing libmbedtls12 (2.28.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libmbedtls12_2.28.5-1_mips_24kc.ipk
Installing hostapd-utils (2023-09-08-e5ccbfc6-5) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/hostapd-utils_2023-09-08-e5ccbfc6-5_mips_24kc.ipk
Installing htop (3.2.2-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/htop_3.2.2-1_mips_24kc.ipk
Installing ip-tiny (6.5.0-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ip-tiny_6.5.0-1_mips_24kc.ipk
Installing iw-full (5.19-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/iw-full_5.19-1_mips_24kc.ipk
Installing iwinfo (2023-07-01-ca79f641-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/iwinfo_2023-07-01-ca79f641-1_mips_24kc.ipk
Installing libiwinfo-data (2023-07-01-ca79f641-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libiwinfo-data_2023-07-01-ca79f641-1_mips_24kc.ipk
Installing libiwinfo20230701 (2023-07-01-ca79f641-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libiwinfo20230701_2023-07-01-ca79f641-1_mips_24kc.ipk
Package kernel (5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd) installed in root is up to date.
Installing kmod (31-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/kmod_31-1_mips_24kc.ipk
Installing zlib (1.3-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/zlib_1.3-1_mips_24kc.ipk
Installing kmod-ath10k-ct (5.15.137+2023-06-05-fadd0768-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-ath10k-ct_5.15.137%2b2023-06-05-fadd0768-1_mips_24kc.ipk
Installing wireless-regdb (2023.09.01-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/wireless-regdb_2023.09.01-1_all.ipk
Installing kmod-cfg80211 (5.15.137+6.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-cfg80211_5.15.137%2b6.5-1_mips_24kc.ipk
Installing kmod-crypto-cmac (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-cmac_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-null (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-null_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-aead (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-aead_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-manager (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-manager_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-hmac (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-hmac_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-sha512 (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-sha512_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-rng (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-rng_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-seqiv (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-seqiv_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-ctr (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-ctr_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-ccm (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-ccm_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-gf128 (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-gf128_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-ghash (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-ghash_5.15.137-1_mips_24kc.ipk
Installing kmod-crypto-gcm (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-crypto-gcm_5.15.137-1_mips_24kc.ipk
Installing kmod-mac80211 (5.15.137+6.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-mac80211_5.15.137%2b6.5-1_mips_24kc.ipk
Installing kmod-ath (5.15.137+6.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-ath_5.15.137%2b6.5-1_mips_24kc.ipk
Installing kmod-hwmon-core (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-hwmon-core_5.15.137-1_mips_24kc.ipk
Installing kmod-ath9k (5.15.137+6.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-ath9k_5.15.137%2b6.5-1_mips_24kc.ipk
Installing kmod-random-core (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-random-core_5.15.137-1_mips_24kc.ipk
Installing kmod-ath9k-common (5.15.137+6.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-ath9k-common_5.15.137%2b6.5-1_mips_24kc.ipk
Package kmod-cfg80211 (5.15.137+6.5-1) installed in root is up to date.
Installing kmod-gpio-button-hotplug (5.15.137-3) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-gpio-button-hotplug_5.15.137-3_mips_24kc.ipk
Package kmod-hwmon-core (5.15.137-1) installed in root is up to date.
Package kmod-mac80211 (5.15.137+6.5-1) installed in root is up to date.
Package kmod-nf-conntrack (5.15.137-1) installed in root is up to date.
Package kmod-nf-conntrack6 (5.15.137-1) installed in root is up to date.
Package kmod-nf-flow (5.15.137-1) installed in root is up to date.
Package kmod-nf-log (5.15.137-1) installed in root is up to date.
Package kmod-nf-log6 (5.15.137-1) installed in root is up to date.
Package kmod-nf-nat (5.15.137-1) installed in root is up to date.
Package kmod-nf-reject (5.15.137-1) installed in root is up to date.
Package kmod-nf-reject6 (5.15.137-1) installed in root is up to date.
Package kmod-nfnetlink (5.15.137-1) installed in root is up to date.
Package kmod-nft-core (5.15.137-1) installed in root is up to date.
Package kmod-nft-fib (5.15.137-1) installed in root is up to date.
Package kmod-nft-nat (5.15.137-1) installed in root is up to date.
Package kmod-nft-offload (5.15.137-1) installed in root is up to date.
Installing kmod-nls-base (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-nls-base_5.15.137-1_mips_24kc.ipk
Installing kmod-phy-ath79-usb (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-phy-ath79-usb_5.15.137-1_mips_24kc.ipk
Installing kmod-usb-core (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-usb-core_5.15.137-1_mips_24kc.ipk
Package kmod-random-core (5.15.137-1) installed in root is up to date.
Installing kmod-scsi-core (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-scsi-core_5.15.137-1_mips_24kc.ipk
Installing kmod-slhc (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-slhc_5.15.137-1_mips_24kc.ipk
Installing kmod-lib-crc-ccitt (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-lib-crc-ccitt_5.15.137-1_mips_24kc.ipk
Package kmod-usb-core (5.15.137-1) installed in root is up to date.
Installing kmod-usb-ehci (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-usb-ehci_5.15.137-1_mips_24kc.ipk
Installing kmod-usb-ledtrig-usbport (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-usb-ledtrig-usbport_5.15.137-1_mips_24kc.ipk
Installing kmod-usb-storage (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-usb-storage_5.15.137-1_mips_24kc.ipk
Installing kmod-usb2 (5.15.137-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/kmods/5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd/kmod-usb2_5.15.137-1_mips_24kc.ipk
Package libiwinfo-data (2023-07-01-ca79f641-1) installed in root is up to date.
Package libiwinfo20230701 (2023-07-01-ca79f641-1) installed in root is up to date.
Installing libkmod (31-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/libkmod_31-1_mips_24kc.ipk
Installing libustream-mbedtls20201210 (2023-02-25-498f6e26-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libustream-mbedtls20201210_2023-02-25-498f6e26-1_mips_24kc.ipk
Installing logd (2022-08-13-4c7b720b-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/logd_2022-08-13-4c7b720b-2_mips_24kc.ipk
Installing lsof (4.94.0-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/lsof_4.94.0-1_mips_24kc.ipk
Installing libpthread (1.2.4-4) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/libpthread_1.2.4-4_mips_24kc.ipk
Installing libtirpc (1.3.3-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/libtirpc_1.3.3-1_mips_24kc.ipk
Installing luci-app-attendedsysupgrade (git-23.292.39411-021c439) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-app-attendedsysupgrade_git-23.292.39411-021c439_all.ipk
Installing rpcd-mod-file (2023-07-01-c07ab2f9-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/rpcd-mod-file_2023-07-01-c07ab2f9-1_mips_24kc.ipk
Installing rpcd-mod-luci (20230123-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/rpcd-mod-luci_20230123-1_mips_24kc.ipk
Installing rpcd-mod-ucode (2023-07-01-c07ab2f9-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/rpcd-mod-ucode_2023-07-01-c07ab2f9-1_mips_24kc.ipk
Installing ucode-mod-math (2023-10-19-07c03173-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/ucode-mod-math_2023-10-19-07c03173-1_mips_24kc.ipk
Installing ucode-mod-html (1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/ucode-mod-html_1_mips_24kc.ipk
Installing liblucihttp0 (2023-03-15-9b5b683f-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/liblucihttp0_2023-03-15-9b5b683f-1_mips_24kc.ipk
Installing liblucihttp-ucode (2023-03-15-9b5b683f-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/liblucihttp-ucode_2023-03-15-9b5b683f-1_mips_24kc.ipk
Installing luci-base (git-23.292.78363-ee6a4da) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-base_git-23.292.78363-ee6a4da_mips_24kc.ipk
Installing luci-app-commands (git-23.267.29502-3e7fe9f) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-app-commands_git-23.267.29502-3e7fe9f_all.ipk
Installing luci-app-firewall (git-23.285.27593-755edf6) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-app-firewall_git-23.285.27593-755edf6_all.ipk
Installing luci-app-opkg (git-23.267.29502-3e7fe9f) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-app-opkg_git-23.267.29502-3e7fe9f_all.ipk
Installing libuclient20201210 (2023-04-13-007d9454-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libuclient20201210_2023-04-13-007d9454-1_mips_24kc.ipk
Installing uclient-fetch (2023-04-13-007d9454-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/uclient-fetch_2023-04-13-007d9454-1_mips_24kc.ipk
Installing opkg (2022-02-24-d038e5b6-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/opkg_2022-02-24-d038e5b6-2_mips_24kc.ipk
Installing luci-app-uhttpd (git-23.267.29502-3e7fe9f) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-app-uhttpd_git-23.267.29502-3e7fe9f_all.ipk
Installing libjson-script20230523 (2023-05-23-75a3b870-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libjson-script20230523_2023-05-23-75a3b870-1_mips_24kc.ipk
Installing uhttpd (2023-06-25-34a8a74d-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/uhttpd_2023-06-25-34a8a74d-1_mips_24kc.ipk
Installing luci-app-upnp (git-23.267.29502-3e7fe9f) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-app-upnp_git-23.267.29502-3e7fe9f_all.ipk
Installing libcap-ng (0.8.3-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/libcap-ng_0.8.3-2_mips_24kc.ipk
Installing librt (1.2.4-4) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/librt_1.2.4-4_mips_24kc.ipk
Installing libuuid1 (2.39.2-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/libuuid1_2.39.2-1_mips_24kc.ipk
Installing miniupnpd-nftables (2.3.3-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/miniupnpd-nftables_2.3.3-2_mips_24kc.ipk
Package luci-base (git-23.292.78363-ee6a4da) installed in root is up to date.
Installing luci-mod-admin-full (git-19.253.48496-3f93650) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-mod-admin-full_git-19.253.48496-3f93650_all.ipk
Installing rpcd-mod-iwinfo (2023-07-01-c07ab2f9-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/rpcd-mod-iwinfo_2023-07-01-c07ab2f9-1_mips_24kc.ipk
Installing luci-mod-status (git-23.298.76573-25dd893) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-mod-status_git-23.298.76573-25dd893_mips_24kc.ipk
Installing luci-mod-system (git-23.243.32495-038fdd9) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-mod-system_git-23.243.32495-038fdd9_all.ipk
Installing luci-mod-network (git-23.292.81782-5d924ed) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-mod-network_git-23.292.81782-5d924ed_all.ipk
Package luci-mod-network (git-23.292.81782-5d924ed) installed in root is up to date.
Package luci-mod-status (git-23.298.76573-25dd893) installed in root is up to date.
Package luci-mod-system (git-23.243.32495-038fdd9) installed in root is up to date.
Installing luci-proto-ipv6 (git-21.148.48881-79947af) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-proto-ipv6_git-21.148.48881-79947af_all.ipk
Installing luci-theme-bootstrap (git-23.249.72972-04d9820) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-theme-bootstrap_git-23.249.72972-04d9820_all.ipk
Installing luci-theme-openwrt-2020 (git-23.207.49566-7cded1f) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/luci-theme-openwrt-2020_git-23.207.49566-7cded1f_all.ipk
Package miniupnpd-nftables (2.3.3-2) installed in root is up to date.
Installing mtd (26) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/mtd_26_mips_24kc.ipk
Installing nano-full (7.2-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/nano-full_7.2-2_mips_24kc.ipk
Package netifd (2023-10-20-5590a80e-2) installed in root is up to date.
Package nftables-json (1.0.9-1) installed in root is up to date.
Installing odhcp6c (2023-05-12-bcd28363-20) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/odhcp6c_2023-05-12-bcd28363-20_mips_24kc.ipk
Installing odhcpd-ipv6only (2023-10-24-d8118f6e-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/odhcpd-ipv6only_2023-10-24-d8118f6e-1_mips_24kc.ipk
Installing openssh-sftp-server (9.5p1-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/openssh-sftp-server_9.5p1-1_mips_24kc.ipk
Package openwrt-keyring (2022-03-25-62471e69-2) installed in root is up to date.
Package opkg (2022-02-24-d038e5b6-2) installed in root is up to date.
Installing procd (2023-06-25-2db83655-3) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/procd_2023-06-25-2db83655-3_mips_24kc.ipk
Installing procd-seccomp (2023-06-25-2db83655-3) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/procd-seccomp_2023-06-25-2db83655-3_mips_24kc.ipk
Installing procd-ujail (2023-06-25-2db83655-3) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/procd-ujail_2023-06-25-2db83655-3_mips_24kc.ipk
Installing procps-ng (3.3.16-3) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/procps-ng_3.3.16-3_mips_24kc.ipk
Installing procps-ng-free (3.3.16-3) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/procps-ng-free_3.3.16-3_mips_24kc.ipk
Installing psmisc (23.4-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/psmisc_23.4-2_mips_24kc.ipk
Installing px5g-mbedtls (9) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/px5g-mbedtls_9_mips_24kc.ipk
Package rpcd (2023-07-01-c07ab2f9-1) installed in root is up to date.
Package rpcd-mod-file (2023-07-01-c07ab2f9-1) installed in root is up to date.
Package rpcd-mod-iwinfo (2023-07-01-c07ab2f9-1) installed in root is up to date.
Package rpcd-mod-luci (20230123-1) installed in root is up to date.
Package rpcd-mod-rpcsys (2023-07-01-c07ab2f9-1) installed in root is up to date.
Installing rpcd-mod-rrdns (20170710) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/luci/rpcd-mod-rrdns_20170710_mips_24kc.ipk
Package rpcd-mod-ucode (2023-07-01-c07ab2f9-1) installed in root is up to date.
Installing sed (4.9-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/sed_4.9-1_mips_24kc.ipk
Installing strace (6.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/strace_6.5-1_mips_24kc.ipk
Installing stubby (0.4.3-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/stubby_0.4.3-1_mips_24kc.ipk
Installing libyaml (0.2.5-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/libyaml_0.2.5-1_mips_24kc.ipk
Installing getdns (1.7.3-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/getdns_1.7.3-2_mips_24kc.ipk
Installing swconfig (12) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/swconfig_12_mips_24kc.ipk
Package terminfo (6.4-2) installed in root is up to date.
Installing uboot-envtools (2023.07.02-2) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/ath79/generic/packages/uboot-envtools_2023.07.02-2_mips_24kc.ipk
Package ubox (2022-08-13-4c7b720b-2) installed in root is up to date.
Package ubus (2023-06-05-f787c97b-1) installed in root is up to date.
Package ubusd (2023-06-05-f787c97b-1) installed in root is up to date.
Installing uci (2023-08-10-5781664d-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/uci_2023-08-10-5781664d-1_mips_24kc.ipk
Package uclient-fetch (2023-04-13-007d9454-1) installed in root is up to date.
Package ucode (2023-10-19-07c03173-1) installed in root is up to date.
Package ucode-mod-fs (2023-10-19-07c03173-1) installed in root is up to date.
Package ucode-mod-html (1) installed in root is up to date.
Package ucode-mod-math (2023-10-19-07c03173-1) installed in root is up to date.
Package ucode-mod-nl80211 (2023-10-19-07c03173-1) installed in root is up to date.
Package ucode-mod-rtnl (2023-10-19-07c03173-1) installed in root is up to date.
Package ucode-mod-ubus (2023-10-19-07c03173-1) installed in root is up to date.
Package ucode-mod-uci (2023-10-19-07c03173-1) installed in root is up to date.
Package ucode-mod-uloop (2023-10-19-07c03173-1) installed in root is up to date.
Package uhttpd (2023-06-25-34a8a74d-1) installed in root is up to date.
Installing uhttpd-mod-ucode (2023-06-25-34a8a74d-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/uhttpd-mod-ucode_2023-06-25-34a8a74d-1_mips_24kc.ipk
Installing urandom-seed (3) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/urandom-seed_3_mips_24kc.ipk
Installing urngd (2023-07-25-7aefb47b-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/base/urngd_2023-07-25-7aefb47b-1_mips_24kc.ipk
Package usign (2020-05-23-f1f65026-1) installed in root is up to date.
Installing wget-ssl (1.21.4-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/mips_24kc/packages/wget-ssl_1.21.4-1_mips_24kc.ipk
Package wireless-regdb (2023.09.01-1) installed in root is up to date.
Package zlib (1.3-1) installed in root is up to date.
Package kmod-random-core (5.15.137-1) installed in root is up to date.
Package base-files (1545-r24315-ae500e62e2) installed in root is up to date.
Package busybox (1.36.1-1) installed in root is up to date.
Package ca-bundle (20230311-1) installed in root is up to date.
Package dnsmasq (2.89-6) installed in root is up to date.
Package dropbear (2022.82-5) installed in root is up to date.
Package firewall4 (2023-09-01-598d9fbb-1) installed in root is up to date.
Package fstools (2023-02-28-bfe882d5-1) installed in root is up to date.
Package kernel (5.15.137-1-75eb6ce3d7c44e9753368a3edec9acfd) installed in root is up to date.
Package kmod-ath10k-ct (5.15.137+2023-06-05-fadd0768-1) installed in root is up to date.
Package kmod-ath9k (5.15.137+6.5-1) installed in root is up to date.
Package kmod-gpio-button-hotplug (5.15.137-3) installed in root is up to date.
Package kmod-nft-offload (5.15.137-1) installed in root is up to date.
Package kmod-usb-ledtrig-usbport (5.15.137-1) installed in root is up to date.
Package kmod-usb2 (5.15.137-1) installed in root is up to date.
Package libc (1.2.4-4) installed in root is up to date.
Package libgcc1 (12.3.0-4) installed in root is up to date.
Package libustream-mbedtls20201210 (2023-02-25-498f6e26-1) installed in root is up to date.
Package logd (2022-08-13-4c7b720b-2) installed in root is up to date.
Package mtd (26) installed in root is up to date.
Package netifd (2023-10-20-5590a80e-2) installed in root is up to date.
Package nftables-json (1.0.9-1) installed in root is up to date.
Package odhcp6c (2023-05-12-bcd28363-20) installed in root is up to date.
Package odhcpd-ipv6only (2023-10-24-d8118f6e-1) installed in root is up to date.
Package opkg (2022-02-24-d038e5b6-2) installed in root is up to date.
Package procd (2023-06-25-2db83655-3) installed in root is up to date.
Package procd-seccomp (2023-06-25-2db83655-3) installed in root is up to date.
Package procd-ujail (2023-06-25-2db83655-3) installed in root is up to date.
Package swconfig (12) installed in root is up to date.
Package uboot-envtools (2023.07.02-2) installed in root is up to date.
Package uci (2023-08-10-5781664d-1) installed in root is up to date.
Package uclient-fetch (2023-04-13-007d9454-1) installed in root is up to date.
Package urandom-seed (3) installed in root is up to date.
Package urngd (2023-07-25-7aefb47b-1) installed in root is up to date.
Configuring kmod-scsi-core.
Configuring kmod-nls-base.
Configuring kmod-usb-core.
Configuring kmod-usb-storage.
Configuring terminfo.
Configuring libnl-tiny1.
Configuring libubox20230523.
Configuring libuci20130104.
Configuring libubus20230605.
Configuring libiwinfo-data.
Configuring libiwinfo20230701.
Configuring iwinfo.
Configuring openssh-sftp-server.
Configuring libatomic1.
Configuring libopenssl3.
Configuring liblucihttp0.
Configuring cgi-io.
Configuring kmod-crypto-gf128.
Configuring libcap-ng.
Configuring libmnl0.
Configuring libpthread.
Configuring librt.
Configuring libuuid1.
Configuring libnftnl11.
Configuring miniupnpd-nftables.
Configuring libuclient20201210.
Configuring uclient-fetch.
Configuring opkg.
Configuring libjson-c5.
Configuring libblobmsg-json20230523.
Configuring rpcd.
Configuring rpcd-mod-file.
Configuring rpcd-mod-luci.
Configuring libucode20230711.
Configuring rpcd-mod-ucode.
Configuring ucode.
Configuring ucode-mod-fs.
Configuring ucode-mod-uci.
Configuring ucode-mod-ubus.
Configuring ucode-mod-math.
Configuring ucode-mod-html.
Configuring liblucihttp-ucode.
Configuring luci-base.
Configuring luci-app-opkg.
Configuring ubusd.
Configuring ubus.
Configuring kmod-crypto-hash.
Configuring kmod-crypto-null.
Configuring kmod-crypto-aead.
Configuring kmod-crypto-manager.
Configuring busybox.
Configuring libncurses6.
Configuring psmisc.
Configuring kmod-crypto-hmac.
Configuring kmod-crypto-sha512.
Configuring kmod-crypto-rng.
Configuring kmod-crypto-seqiv.
Configuring kmod-crypto-ctr.
Configuring libtirpc.
Configuring lsof.
Configuring kmod-nfnetlink.
Configuring kmod-nf-reject.
Configuring kmod-nf-reject6.
Configuring kmod-nf-conntrack.
Configuring kmod-nf-conntrack6.
Configuring kmod-nf-nat.
Configuring kmod-nf-log.
Configuring kmod-nf-log6.
Configuring kmod-crypto-crc32c.
Configuring kmod-lib-crc32c.
Configuring kmod-nft-core.
Configuring kmod-nft-fib.
Configuring getdns.
Configuring nano-full.
Configuring swconfig.
Configuring strace.
Configuring luci-mod-system.
Configuring kmod-nf-flow.
Configuring kmod-lib-crc-ccitt.
Configuring ucode-mod-uloop.
Configuring libmbedtls12.
Configuring px5g-mbedtls.
Configuring getrandom.
Configuring dmesg.
Configuring luci-theme-bootstrap.
Configuring iw-full.
Configuring procd-ujail.
Configuring jshn.
Configuring netifd.
Configuring jsonfilter.
Configuring usign.
Configuring openwrt-keyring.
Configuring ubox.
Configuring fstools.
Configuring fwtool.
Configuring base-files.
Configuring libpcre2.
Configuring grep.
Configuring libustream-mbedtls20201210.
Configuring libjson-script20230523.
Configuring uhttpd.
Configuring luci-app-uhttpd.
Configuring kmod-nft-nat.
Configuring kmod-nft-offload.
Configuring jansson4.
Configuring nftables-json.
Configuring firewall4.
Configuring uboot-envtools.
Configuring dnsmasq.
Configuring procd.
Configuring kmod-phy-ath79-usb.
Configuring zlib.
Configuring libkmod.
Configuring hostapd-utils.
Configuring kmod-usb-ehci.
Configuring rpcd-mod-iwinfo.
Configuring luci-mod-status.
Configuring block-mount.
Configuring kmod-usb2.
Configuring sed.
Configuring luci-app-firewall.
Configuring ethtool-full.
Configuring odhcp6c.
Configuring procps-ng.
Configuring procps-ng-free.
Configuring kmod-crypto-ghash.
Configuring wireless-regdb.
Configuring kmod-cfg80211.
Configuring kmod-crypto-cmac.
Configuring kmod-crypto-ccm.
Configuring kmod-crypto-gcm.
Configuring hostapd-common.
Configuring kmod-mac80211.
Configuring kmod-ath.
Configuring kmod-random-core.
Configuring kmod-ath9k-common.
Configuring kmod-ath9k.
Configuring uci.
Configuring dropbear.
Configuring kmod-hwmon-core.
Configuring libyaml.
Configuring ca-bundle.
Configuring stubby.
Configuring luci-theme-openwrt-2020.
Configuring mtd.
Configuring odhcpd-ipv6only.
Configuring procd-seccomp.
Configuring ucode-mod-nl80211.
Configuring urandom-seed.
Configuring luci-mod-network.
Configuring luci-mod-admin-full.
Configuring luci-app-commands.
Configuring kmod-gpio-button-hotplug.
Configuring logd.
Configuring kmod.
Configuring luci-proto-ipv6.
Configuring rpcd-mod-rpcsys.
Configuring attendedsysupgrade-common.
Configuring luci-app-attendedsysupgrade.
Configuring libldns.
Configuring drill.
Configuring ip-tiny.
Configuring ucode-mod-rtnl.
Configuring hostapd-mbedtls.
Configuring luci-app-upnp.
Configuring ath10k-board-qca988x.
Configuring htop.
Configuring kmod-ath10k-ct.
Configuring kmod-usb-ledtrig-usbport.
Configuring wget-ssl.
Configuring urngd.
Configuring kmod-slhc.
Configuring uhttpd-mod-ucode.
Configuring rpcd-mod-rrdns.

Finalizing root filesystem...
/home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/etc/init.d/dnsmasq: line 4: /lib/functions.sh: No such file or directory
Enabling boot
Enabling bootcount
Enabling cron
./etc/init.d/dnsmasq: line 4: /lib/functions.sh: No such file or directory
Enabling dnsmasq
Enabling done
Enabling dropbear
Enabling firewall
Enabling fstab
Enabling gpio_switch
Enabling led
Enabling log
Enabling miniupnpd
Enabling network
Enabling odhcpd
Enabling packet_steering
Enabling radius
Enabling rpcd
Enabling stubby
Enabling sysctl
Enabling sysfixtime
Enabling sysntpd
Enabling system
Enabling ucitrack
Enabling uhttpd
Enabling umount
Enabling urandom_seed
Enabling urngd
Enabling wpad

Building images...
Parallel mksquashfs: Using 12 processors
Creating 4.0 filesystem on /home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/linux-ath79_generic/root.squashfs, block size 262144.
Pseudo file "dev" exists in source filesystem "/home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ath79/dev".
Ignoring, exclude it (-e/-ef) to override.
[=========================================================================================================================================-] 1309/1309 100%

Exportable Squashfs 4.0 filesystem, xz compressed, data block size 262144
        compressed data, compressed metadata, compressed fragments,
        no xattrs, compressed ids
        duplicates are removed
Filesystem size 6654.53 Kbytes (6.50 Mbytes)
        26.71% of uncompressed filesystem size (24911.87 Kbytes)
Inode table size 11290 bytes (11.03 Kbytes)
        20.41% of uncompressed inode table size (55321 bytes)
Directory table size 16026 bytes (15.65 Kbytes)
        44.09% of uncompressed directory table size (36348 bytes)
Number of duplicate files found 190
Number of inodes 1667
Number of files 1280
Number of fragments 64
Number of symbolic links 248
Number of device nodes 1
Number of fifo nodes 0
Number of socket nodes 0
Number of directories 138
Number of hard-links 0
Number of ids (unique uids + gids) 1
Number of uids 1
        unknown (0)
Number of gids 1
        unknown (0)
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-ath79-generic-tplink_archer-c7-v2-squashfs-sysupgrade.bin.new" completed
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-ath79-generic-tplink_archer-c7-v2-squashfs-factory.bin.new" completed
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-ath79-generic-tplink_archer-c7-v2-squashfs-factory-us.bin.new" completed
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-ath79-generic-tplink_archer-c7-v2-squashfs-factory-eu.bin.new" completed
Cannot open '/home/feig/openwrt-imagebuilder-ath79-generic.Linux-x86_64/tmp/.packageinfo': No such file or directory
make[3]: *** [Makefile:104: install] Error 1
make[2]: *** [Makefile:208: build_image] Error 2
make[1]: *** [Makefile:146: _call_image] Error 2
make: *** [Makefile:262: image] Error 2
feig@host ~/openwrt-imagebuilder-ath79-gen

### OpenWrt version

main

### OpenWrt target/subtarget

ath79/generic

### Device

PC imagebuilder on Linux amd64 for ath79/generic for c7v2

### Image kind

Official downloaded image

### Steps to reproduce

see log

### Actual behaviour

see log

### Expected behaviour

.packageinfo error not existing

### Additional info

I used ath79 generic image builder download from latest snapshot. I extracted with tar xvJf then ran command in logs.
btw attended sysupgrade doesn't work either. so that's 2/2 of usual avenues to make this image not working and I'll try the 3rd after.
for asu, here's the log:
(I'll edit it in here but basically it flails bc no /lib/functions.sh)
here:
list of available packages in /builder/build_dir/target-mips_24kc_musl/root-ath79/../../../../builder/dl/imagebuilder Downloading file:packages/Packages.sig Signature check passed. /builder/build_dir/target-mips_24kc_musl/root-ath79/etc/init.d/dnsmasq: line 4: /lib/functions.sh: No such file or directory ./etc/init.d/dnsmasq: line 4: /lib/functions.sh: No such file or directory Pseudo file "dev" exists in source filesystem "/builder/build_dir/target-mips_24kc_musl/root-ath79/dev". Ignoring, exclude it (-e/-ef) to override. [mktplinkfw] rootfs offset aligned to 0x2484536 [mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-9c28eb6c715c-ath79-generic-tplink_archer-c7-v2-squashfs-sysupgrade.bin.new" completed [mktplinkfw] rootfs offset aligned to 0x2484536 [mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-9c28eb6c715c-ath79-generic-tplink_archer-c7-v2-squashfs-factory.bin.new" completed [mktplinkfw] rootfs offset aligned to 0x2484536 [mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-9c28eb6c715c-ath79-generic-tplink_archer-c7-v2-squashfs-factory-us.bin.new" completed [mktplinkfw] rootfs offset aligned to 0x2484536 [mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-9c28eb6c715c-ath79-generic-tplink_archer-c7-v2-squashfs-factory-eu.bin.new" completed Cannot open '/builder/tmp/.packageinfo': No such file or directory make[3]: *** [Makefile:104: install] Error 1 make[2]: *** [Makefile:208: build_image] Error 2 make[1]: *** [Makefile:146: _call_image] Error 2 make: *** [Makefile:262: image] Error 2 Close

### Diffconfig

_No response_

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

ok for 3/3 firmware selector also fales:
ownloading file:packages/Packages
Updated list of available packages in /builder/build_dir/target-mips_24kc_musl/root-ath79/../../../../builder/dl/imagebuilder
Downloading file:packages/Packages.sig
Signature check passed.
/builder/build_dir/target-mips_24kc_musl/root-ath79/etc/init.d/dnsmasq: line 4: /lib/functions.sh: No such file or directory
./etc/init.d/dnsmasq: line 4: /lib/functions.sh: No such file or directory
Pseudo file "dev" exists in source filesystem "/builder/build_dir/target-mips_24kc_musl/root-ath79/dev".
Ignoring, exclude it (-e/-ef) to override.
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-735586dcbae6-ath79-generic-tplink_archer-c7-v2-squashfs-sysupgrade.bin.new" completed
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-735586dcbae6-ath79-generic-tplink_archer-c7-v2-squashfs-factory.bin.new" completed
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-735586dcbae6-ath79-generic-tplink_archer-c7-v2-squashfs-factory-us.bin.new" completed
[mktplinkfw] rootfs offset aligned to 0x2484536
[mktplinkfw] firmware file "/builder/build_dir/target-mips_24kc_musl/linux-ath79_generic/tmp/openwrt-735586dcbae6-ath79-generic-tplink_archer-c7-v2-squashfs-factory-eu.bin.new" completed
Cannot open '/builder/tmp/.packageinfo': No such file or directory
make[3]: *** [Makefile:104: install] Error 1
make[2]: *** [Makefile:208: build_image] Error 2
make[1]: *** [Makefile:146: _call_image] Error 2
make: *** [Makefile:262: image] Error 2

--- a/include/image.mk
+++ b/include/image.mk
@@ -277,11 +277,13 @@ endef
 define Image/Manifest
 	$(call opkg,$(TARGET_DIR_ORIG)) list-installed > \
 		$(BIN_DIR)/$(IMG_PREFIX)$(if $(PROFILE_SANITIZED),-$(PROFILE_SANITIZED)).manifest
+ifndef IB
 	$(if $(CONFIG_JSON_CYCLONEDX_SBOM), \
 		$(SCRIPT_DIR)/package-metadata.pl imgcyclonedxsbom \
 		$(TMP_DIR)/.packageinfo \
 		$(BIN_DIR)/$(IMG_PREFIX)$(if $(PROFILE_SANITIZED),-$(PROFILE_SANITIZED)).manifest > \
 		$(BIN_DIR)/$(IMG_PREFIX)$(if $(PROFILE_SANITIZED),-$(PROFILE_SANITIZED)).bom.cdx.json)
+endif
 endef
 
 define Image/gzip-ext4-padded-squashfs

